.github/workflows/pr-check-license.yaml

```diff
@@ -0,0 +1,29 @@
+name: Check License
```

Comment:
Keep the name in same format with other file

Reply:
Fixed.